src/main/resources/swagger/api-docs.yaml

```diff
@@ -1606,6 +1606,121 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/resources/v2/{resourceTypeName}/{resourceId}/parent:
```

Comment:
Do we also need to add parent/child info to getWorkspace?

Reply:
Hmm, good question. I think that parent/child info from Sam could eventually be bubbled up to the Rawls `getWorkspace` endpoint once Rawls has been updated to use hierarchical resources in Sam. I'm not sure about that though, it will depend on how Rawls ends up using them. At the moment, I could imagine hierarchical resources in Sam being used to track a workspace's namespace as in the parent of all `workspace` resources in Sam are some `billing-project` resource. However, I think Rawls would keep track of that parent/child relationship itself as it already tracks workspace namespaces in its own database. Sam's parent/child knowledge in that case would just be used to verify permissions, I don't think Rawls would necessarily ask Sam to find out which resource is the parent of a workspace. All that said, the `getWorkspace` endpoint in Rawls will eventually be a passthrough to Workspace Manager (I think) and I have no idea how it'll be implemented there.

Reply:
ok -- thanks for the detailed explanation!